src/config.rs

```diff
@@ -611,3 +655,71 @@ fn test_relative_path_resolution() {
         Path::new("/path/to/config/../custom_pages")
     );
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    mod language {
+        use super::*;
+
+        #[test]
+        fn missing_lang_env() {
+            let lang_list = get_languages(None, Some("de:fr"));
+            assert_eq!(lang_list, [Language("en")]);
```

Comment:
A stupid question. Why are `de` and `fr` missing here? I know it's copy & paste code.